components/eventing-controller/pkg/env/nats_config.go

```diff
@@ -36,6 +42,9 @@ type NatsConfig struct {
 	JSStreamRetentionPolicy string `envconfig:"JS_STREAM_RETENTION_POLICY" default:"interest"`
 	JSStreamMaxMessages     int64  `envconfig:"JS_STREAM_MAX_MSGS" default:"-1"`
 	JSStreamMaxBytes        int64  `envconfig:"JS_STREAM_MAX_BYTES" default:"-1"`
+
+	// Name of the JetStream stream where all events are stored.
+	JSStreamName string `default:"kyma"`
```

Comment:
Please mark this as required too.

Reply:
The value of `JSStreamName` will not be provided in the envs so it cannot be marked as required. Instead it will set based on the `eventTypePrefix` [here](https://github.com/kyma-project/kyma/blob/37ca1cd3c36cdd527cf69a6bcf7bb1294c759cb0/components/eventing-controller/pkg/env/nats_config.go#L60). If the `eventTypePrefix` is empty, then it will use the `default:"kyma"`

Reply:
Its outdated now.